packages/api/schema/stryker-core.json

```diff
@@ -168,18 +168,30 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "fileHeaders": {
+        "ignorePatterns": {
+          "description": "Ignore these patterns when preprocessing the sandbox files",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "default": [
+            "**/__tests__/**/*.{ts,tsx}",
+            "**/*.{spec,test}.{ts,tsx}",
+            "**/*{Spec,Test}.{ts,tsx}"
+          ]
+        },
+        "addHeaders": {
           "type": "object",
           "title": "SandboxFileHeaders",
           "description": "Configure additional headers to be added to files inside your sandbox. These headers will be added after Stryker has instrumented your code with mutants, but before a test runner or build command is executed. This is used to ignore typescript compile errors and eslint warnings that might have been added in the process of instrumenting your code with mutants. The default setting should work for most use cases.",
           "additionalProperties": {
             "type": "string"
           },
           "default": {
-            "**/*+(.js|.ts|.cjs|.mjs)?(x)": "/* eslint-disable */\n// @ts-nocheck\n"
+            "**/*.{ts,tsx}": "// @ts-nocheck\n"
           }
         },
-        "stripComments": {
+        "removeComments": {
```

Comment:
So `remove` is better than `strip`? I guess I've used the naming of the `strip-comments` dependency. Remove sounds better indeed.